.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Cargo fmt all code
+c11ef36365f58f0b107c578ec9219ee1c86b09d2
```

Comment:
We usually merge PR by rebase or squash, so the commits merged into master will be different commit. Could you separate this into another PR?

Reply:
sure thing, I'll make the PR once this gets merged